src/react/actions/__tests__/s3bucket.test.jsx

```diff
@@ -15,6 +16,16 @@ describe('s3bucket actions', () => {
             fn: actions.listBucketsSuccess([], OWNER_NAME),
             expectedActions: [dispatchAction.LIST_BUCKETS_SUCCESS_ACTION([], OWNER_NAME)],
         },
+        {
+            it: 'should return OPEN_BUCKET_DELETE_DIALOG action and bucket name',
+            fn: actions.openBucketDeleteDialog(''),
```

Comment:
Use a non-empty bucketName to make the test more "realistic".

Reply:
done: I create a BUCKET_NAME var to make other part of test much more cleaner too